modules/openlmis-web/src/main/webapp/public/js/distribution/model/adult-coverage.js

```diff
@@ -20,6 +20,38 @@ function AdultCoverage(facilityVisitId, adultCoverageJSON) {
   $(this.openedVialLineItems).each(function (i, lineItem) {
     _this.openedVialLineItems[i] = new OpenedVialLineItem(lineItem);
   });
+
+  var fieldList = ['healthCenterTetanus1', 'outreachTetanus1', 'healthCenterTetanus2To5', 'outreachTetanus2To5'];
+
+  function init() {
+    var countNotNR = 0;
+    var countNR = 0;
+    $(this.adultCoverageLineItems).each(function (i, lineItem) {
+      $(fieldList).each(function (i, fieldName) {
+        if(isUndefined(lineItem[fieldName]) || lineItem[fieldName].notRecorded === false) {
+            countNotNR++;
+        }
+        else if(lineItem[fieldName].notRecorded === true) {
+            countNR++;
+        }
+      });
+    });
+    $(this.openedVialLineItems).each(function (i, lineItem) {
+      if(isUndefined(lineItem.openedVial) || lineItem.openedVial.notRecorded === false) {
+          countNotNR++;
+      }
+      else if(lineItem.openedVial.notRecorded === true) {
+          countNR++;
+      }
+    });
```

Comment:
This looks to work and makes sense. I wonder, though, whether we can reduce the code (and maintenance cost) by letting either jQuery or native selectors do the work. For example, if each NR checkbox were given an attribute of “semanticType” set to “nrCheckbox,” it seems we could replace the above block of code with something like:

countNR = $("[ semanticType = nrCheckbox]").is(':checked')
countNotNR = $("[ semanticType = nrCheckbox]").not(':checked')

Does that seem reasonable to you?